designer/server/src/routes/forms/create.js

```diff
@@ -72,10 +76,35 @@ export default [
     options: {
       validate: {
         payload: Joi.object().keys({
-          title: titleSchema.messages({
-            'string.empty': 'Enter your form name',
-            'string.max': 'Form name must be 250 characters or less'
-          })
+          title: titleSchema
+            .external(
```

Comment:
I would recommend avoiding `joi.external` and do any "2nd phase" validation logic like this in the main handler.

Reply:
Ah can you explain why?

Only because we've got to validate the entire metadata before submission soon:

* https://github.com/DEFRA/forms-designer/pull/150

Putting it into a schema gives us a tidy way to limit error handling to `failAction()`